### PR TITLE
2023.11.0

### PR DESCRIPTION
**Pull request by frenck**
⚠️ This PR is for reviewing purposes. Please do not merge!

### CODE REVIEW

File: source/_posts/2023-11-01-release-202311.markdown [hunk 1]
```diff
@@ -0,0 +1,637 @@
...
+Don't forget to [join our release party live stream on YouTube](https://www.youtube.com/watch?v=QlpC0LFM8l8)
+1 November 2023, at 13:00 PDT / 21:00 CET!
+
+<lite-youtube videoid="QlpC0LFM8l8" videotitle="Home Assistant 2023.11 Release Party"></lite-youtube>
+
+## There is a lot to-do!
+
+There are many things to do in our lives, which is what
+[@allenporter] realized too! We now have a new entity type
+in Home Assistant: [`todo` entities](/integrations/todo)!
```
Comment (MartinHjelmare) [inline source/_posts/2023-11-01-release-202311.markdown]: Should we call them To-do list entities as in our docs or just todo entities?

File: source/_posts/2023-11-01-release-202311.markdown [hunk 2]
```diff
@@ -0,0 +1,637 @@
...
+dashboards, now fittingly named **To-do list card**, to support multiple
+to-do lists. You can now select which to-do list you want to show on the card
+you've put in your dashboard.
+
+<img class="no-shadow" src='/images/blog/2023-11/todo-list-card.png' alt='Screenshot showing configuration of a to-do list card, that allows selecting the to-do list to display.'>
+
+## Integrations providing your to-dos
+
+But wait! There is more besides local to-do lists and shopping lists.
+Now we have a to-do entity, we can also start using it with other integrations
```
Comment (MartinHjelmare) [inline source/_posts/2023-11-01-release-202311.markdown]:
```suggestion
Now that we have a to-do entity, we can also start using it with other integrations
```

**Review by janiversen — DISMISSED** [review]

File: source/_posts/2023-11-01-release-202311.markdown [hunk 3]
```diff
@@ -0,0 +1,637 @@
...
+### User condition <!-- omit in toc -->
+
+The user condition allows you to show or hide cards based on the user who is
+currently logged in. Useful if you have multiple users in your Home Assistant
+instance and want to show or hide cards based on who is looking at the
+dashboard.
+
+<img class="no-shadow" src='/images/blog/2023-11/conditional-card-per-user.png' alt='Screenshot showing the configuration dialog of the conditional card, which now allows you to set which users to show the card at.'>
+
+The screenshot above shows how the card is only shown when the user is
```
Comment (MartinHjelmare) [inline source/_posts/2023-11-01-release-202311.markdown]:
```suggestion
The screenshot above shows how the card is only shown when the user
```

File: source/_posts/2023-11-01-release-202311.markdown [hunk 4]
```diff
@@ -0,0 +1,637 @@
...
+who is logged in is Frenck. It is hidden for all other users.
+
+### Numeric state condition <!-- omit in toc -->
+
+Similar to the state condition, the numeric state condition allows you to show
+or hide cards based on the state of a entity, only this time, based on its
```
Comment (MartinHjelmare) [inline source/_posts/2023-11-01-release-202311.markdown]:
```suggestion
or hide cards based on the state of an entity, only this time, based on its
```

File: source/_posts/2023-11-01-release-202311.markdown [hunk 5]
```diff
@@ -0,0 +1,637 @@
...
+numeric value. Useful if you want to show or hide cards based on temperature,
+humidity, or any other numeric entity.
+
+<img class="no-shadow" src='/images/blog/2023-11/conditional-card-numeric-state.png' alt='Screenshot showing the configuration dialog of the conditional card, which you can now configure an numeric condition on when to show the card.'>
```
Comment (MartinHjelmare) [inline source/_posts/2023-11-01-release-202311.markdown]:
```suggestion
<img class="no-shadow" src='/images/blog/2023-11/conditional-card-numeric-state.png' alt='Screenshot showing the configuration dialog of the conditional card, on which you can now configure a numeric condition when to show the card.'>
```

File: source/_posts/2023-11-01-release-202311.markdown [hunk 6]
```diff
@@ -0,0 +1,637 @@
...
+
+### Screen condition <!-- omit in toc -->
+
+Last but not least, the screen condition allows you to show or hide cards based
+on the screen size of the device you are viewing Home Assistant on.
+
+For example, you could show some cards only when viewing Home Assistant on a
+desktop or tablet and hide them when viewing Home Assistant on a mobile device.
+
+<img class="no-shadow" src='/images/blog/2023-11/conditional-card-screen-type.png' alt='Screenshot showing the configuration dialog of the conditional card, which you can now configure an numeric condition on when to show the card.'>
```
Comment (MartinHjelmare) [inline source/_posts/2023-11-01-release-202311.markdown]: Alt text is copy paste from the numeric state condition.

File: source/_posts/2023-11-01-release-202311.markdown [hunk 7]
```diff
@@ -0,0 +1,637 @@
...
+it and help you set it up by connecting it to your Wi-Fi network and adding 
+it to Home Assistant.
+
+<img class="no-shadow" src='/images/blog/2023-11/improv-wifi.png' alt='Screenshot showing a discovered Improv Wi-Fi device over Bluetooth, which can be set up and added to your Wi-Fi network straight from Home Assistant.'>
+
+[Improv Wi-Fi](https://improv-wifi.com/) is a fully open standard that can
+be freely adopted and implemented by any hardware creator, no matter how big
+or small. Improv Wi-Fi is also supported by [ESPHome](https://esphome.io/components/esp32_improv#improv-via-ble).
+
+Nice job on this one [@emonemery]!
```
Comment (MartinHjelmare) [inline source/_posts/2023-11-01-release-202311.markdown]:
```suggestion
Nice job on this one [@emontnemery]!
```

File: source/_posts/2023-11-01-release-202311.markdown [hunk 8]
```diff
@@ -0,0 +1,637 @@
...
+specific person and use a field to define the message you want to send.
+
+These fields support [our UI selectors](/docs/blueprint/selectors/), so you
+can provide a UI for your script fields. However, this was only available
+when you created your scripts in YAML format.
+
+But not anymore! Thanks to [@karwosts], you can now also configure your
+script fields in the UI!
+
+<img class="no-shadow" src='/images/blog/2023-11/script-fields.png' alt='Screenshot showing the scripts fields in the UI while editting a script in the script editor.'>
```
Comment (MartinHjelmare) [inline source/_posts/2023-11-01-release-202311.markdown]:
```suggestion
<img class="no-shadow" src='/images/blog/2023-11/script-fields.png' alt='Screenshot showing the scripts fields in the UI while editing a script in the script editor.'>
```

File: source/_posts/2023-11-01-release-202311.markdown [hunk 9]
```diff
@@ -0,0 +1,637 @@
...
+[@allenporter]: https://github.com/allenporter
+[@emonemery]: https://github.com/emontnemery
+[@Guy293]: https://github.com/Guy293
+[Google Tasks]: /integrations/google_tasks
+[Improv BLE]: /integrations/improv_ble
+[Local to-do list]: /integrations/local_todo
+[Tami4 Edge / Edge+]: /integrations/tami4
+[To-do list]: /integrations/todo
+
+This release also has a new virtual integration. Virtual integrations are stubs that are handled by other (existing) integrations to help with findability. The following virtual integration has been added:
```
Comment (MartinHjelmare) [inline source/_posts/2023-11-01-release-202311.markdown]:
```suggestion
This release also has a new virtual integration. Virtual integrations are stubs that are handled by other (existing) integrations to help with findability. The following virtual integrations have been added:
```

File: source/_posts/2023-11-01-release-202311.markdown [hunk 10]
```diff
@@ -0,0 +1,637 @@
...
+
+# This is correct
+demo:
+
+# This no longer works and should be removed
+light:
+  - platform: demo
+```
+
+([@gjohansson-ST] - [#100867]) ([documentation](/integrations/homekit))
```
Comment (MartinHjelmare) [inline source/_posts/2023-11-01-release-202311.markdown]: Documentation link is incorrect.

Comment (MartinHjelmare) [reply source/_posts/2023-11-01-release-202311.markdown]:
```suggestion
([@gjohansson-ST] - [#100867]) ([documentation](/integrations/demo))
```

**Pull request merged**